### PR TITLE
Add a button to swap main and other user in user merge view

### LINKED ISSUE #1781: Swapping users in user profile merge preview
On the user profile merge preview page, the two previously selected user profiles are shown in preparation for merging the two user profiles.
On the top right of the page, a new button should be added, reloading the page with the main user and other user swapped.

### PR DESCRIPTION
**Pull request by postmartem**
Closes #1781

### CODE REVIEW

File: evap/staff/templates/staff_user_merge.html [hunk 1]
```diff
@@ -10,7 +10,12 @@
 
 {% block content %}
     {{ block.super }}
-    <h3>{% trans 'Merge users' %}</h3>
+    <div class="d-flex">
+        <h3>{% trans 'Merge users' %} </h3> 
+        <div class="ms-auto"> 
+            <a class="btn btn-sm btn-light" href="{% url 'staff:user_merge' main_user_id=other_user.pk other_user_id=main_user.pk %}">{% trans 'Swap main user' %}  </a>
+        </div>
```
Comment (niklasmohrin) [inline evap/staff/templates/staff_user_merge.html:17]: Nit: There is some extra whitespace here
```suggestion
        <h3>{% trans 'Merge users' %}</h3> 
        <div class="ms-auto"> 
            <a class="btn btn-sm btn-light" href="{% url 'staff:user_merge' main_user_id=other_user.pk other_user_id=main_user.pk %}">{% trans 'Swap main user' %}</a>
        </div>
```

**Review by niklasmohrin — APPROVED** [review]

**Review by He3lixxx — APPROVED** [review]
Nice, thanks! :)

File: evap/staff/templates/staff_user_merge.html [hunk 2]
```diff
@@ -10,7 +10,12 @@
 
 {% block content %}
     {{ block.super }}
-    <h3>{% trans 'Merge users' %}</h3>
+    <div class="d-flex">
+        <h3>{% trans 'Merge users' %}</h3> 
+        <div class="ms-auto"> 
+            <a class="btn btn-sm btn-light" href="{% url 'staff:user_merge' main_user_id=other_user.pk other_user_id=main_user.pk %}">{% trans 'Swap main user' %}</a>
```
Comment (janno42) [inline evap/staff/templates/staff_user_merge.html:16]:
```suggestion
            <a class="btn btn-sm btn-light" href="{% url 'staff:user_merge' main_user_id=other_user.pk other_user_id=main_user.pk %}">{% trans 'Swap users' %}</a>
```

**Review by janno42 — CHANGES_REQUESTED** [review]

**Review by janno42 — APPROVED** [review]

**Comment by janno42** [comment]
✔️ Meets requirements
✔️ UI functionality checked

**Pull request merged**